source/scripts/JobDetails.js

```diff
@@ -68,47 +208,100 @@ class JobDetails extends HTMLElement {
     // Set the contents of the <details> with the <details> template
     // given in job-details.html and the data passed in
     details.innerHTML = `
-      <summary class="summary">
-        <div id="company">${data.company}</div>
-        <div id="position-title">${data.title}</div>
-        <div id="location">${data.location}</div>
-        <div id="status">${data.status}</div>
-        <div id="deadline-date">${data.deadlineDate}</div>
-        <div id="deadline-time">${data.deadlineTime}</div>
+      <summary>
+        <div id="arrow"><img src="./assets/next.png"></div>
+        <div id="company">${data.company}</div> 
+        <div id="title">${data.position}</div>
+        <div id="location-tag">
+          <span id="pin-symbol">📍</span>
+          <span id="location">${data.location}</span>
+        </div>
+        <div id="status">Status: ${data.status}</div>
+        <div id="notification-symbol">🔔</div>
+        <div id="deadline">
+          <span id="deadline-date">MM/DD/YY</span>
+          <span id="deadline-time">HH:MM XM</span>
+        </div>
       </summary>
-      <div>
-        <h1>Description</h1>
-        <p>${data.description}</p>
-      </div>
-      <div>
-        <h1>Notes</h1>
-        <ul id="notes">
-        </ul>
-      </div>
-      <h1>Contact</h1>
-      <a href="${data.contact}">${data.contact}</a>
-      <h1>Portal</h1>
-      <a href="${data.portal}">${data.portal}</a>
-      <h1>Tags</h1>
-      <div id="tags">
+      <div class="dropdown">
+        <hr id="horizontal-rule">
+        <div class="heading">Description</div>
```

Comment:
add some ternaries for some default text that shows up if fields are empty

Reply:
Would it be better for the heading for that field to not show up when the field is empty?
Here is an before/after example for an empty "Notes" field:

![image](https://user-images.githubusercontent.com/97627312/204575131-d30c1add-7ff7-477e-acda-d2ca57469352.png)

![image](https://user-images.githubusercontent.com/97627312/204575367-64acc08c-48fb-47bc-8a57-77c12d515780.png)

As opposed to:

![image](https://user-images.githubusercontent.com/97627312/204576131-b2be7a9b-f8da-46aa-947f-22c7c568d1ad.png)

Reply:
Just added the change:

![image](https://user-images.githubusercontent.com/97627312/204657815-7e2874a7-aa4e-41bd-a6ee-23560d211d31.png)